core-web/libs/dotcms-scss/shared/_colors.scss

```diff
@@ -226,6 +226,16 @@ $success: $color-accessible-text-green;
     --color-palette-primary-800: hsl(var(--color-primary-h) var(--color-primary-s) 27%);
     --color-palette-primary-900: hsl(var(--color-primary-h) var(--color-primary-s) 21%);
 
+    --primary-100: hsl(var(--color-primary-h) var(--color-primary-s) 98%);
```

Comment:
From where do these colors come from?

Reply:
Those colors are a copy of --color-palette-primary-100. I fixed it to avoid confusion. In summary, the idea is to use the PrimeFlex classes, such as text-primary-500, to avoid repeatedly creating styles like this:


.myclass {
    color: $color-palette-primary-500;
}

Instead, define it like this: <p class="text-primary-500"></p>.

The class below represents repetitive code, which can be avoided by using the text-color class provided by PrimeFlex. However, these classes do not work because they are not defined in our theme.